src/main/java/com/xjeffrose/xio/SSL/X509CertificateGenerator.java

```diff
@@ -58,14 +58,14 @@ public static DERKeySpec parseDERKeySpec(Path path) {
     return parseDERKeySpec(rawKeyString);
   }
 
-    public static DERKeySpec parseDERKeySpec(String rawKeyString) {
+  public static DERKeySpec parseDERKeySpec(String rawKeyString) {
     try {
       // Base64 decode the data
       Base64.Decoder b64decoder = Base64.getDecoder();
       byte[] encoded = b64decoder.decode(
-          rawKeyString.replace("-----BEGIN RSA PRIVATE KEY-----\n", "")
-              .replace("-----END RSA PRIVATE KEY-----\n", "")
-              .replace("\n", "")
+        rawKeyString.replace("-----BEGIN RSA PRIVATE KEY-----", "")
+          .replace("-----END RSA PRIVATE KEY-----", "")
+          .replace(System.getProperty("line.separator"), "")
```

Comment:
Should this be two statements "\n" -> "" and "\r" -> "" instead to make this work even when a unix file ends up on windows or vice versa?